packages/shortcode/src/index.js

```diff
@@ -143,9 +143,9 @@ export function string( options ) {
  *
  * @return {RegExp} Shortcode RegExp.
  */
-export const regexp = memize( ( tag ) => {
+export function regexp( tag ) {
 	return new RegExp( '\\[(\\[?)(' + tag + ')(?![\\w-])([^\\]\\/]*(?:\\/(?!\\])[^\\]\\/]*)*?)(?:(\\/)\\]|\\](?:([^\\[]*(?:\\[(?!\\/\\2\\])[^\\[]*)*)(\\[\\/\\2\\]))?)(\\]?)', 'g' );
-} );
+};
```

Comment:
Semi-colon should be removed.

Reply:
I wouldn't hate you if you just removed it yourself :) a712bf7eb439f3f398dcfb7e6f850222c1e7cb8c